src/libraries/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -5217,11 +5217,23 @@ public abstract partial class X86Base
         internal X86Base() { }
         public static bool IsSupported { get { throw null; } }
         public static (int Eax, int Ebx, int Ecx, int Edx) CpuId(int functionId, int subFunctionId) { throw null; }
+        [System.Runtime.Versioning.RequiresPreviewFeaturesAttribute("DivRem is in preview.")]
```

Comment:
Is the plan that we ship these in .NET 8 as preview, or is the plan that by the time we ship .NET 8 it's fully supported?

Reply:
Yes, we plan to address it in .NET 8. Tracking issue https://github.com/dotnet/runtime/issues/82194

Reply:
Ok.  Technically every new API we add in previews is preview and doesn't require such attribution; this attribute was really intended to be used for things to remain in preview in supported releases.  If we're just trying to scare people off from using it in previews, I guess it's ok, but it seems superfluous to me.

Reply:
I agree that this would be superfluous for ordinary APIs. 

This is codegen intrinsic that may require non-trivial work to complete. Tagging it as preview makes us "ready to ship" without any additional changes, even in the case we do not have a chance to complete the work for some reason. Deleting the tag once the work is complete is a small clean change.

Reply:
Fair enough.  Of course, we still need to remember to remove the attribute when we complete that additional work.  Let's make sure to track that somewhere.

Reply:
Updated https://github.com/dotnet/runtime/issues/82194 to capture it.